plugins/avatar/client/src/components/Chat.jsx

```diff
@@ -21,6 +22,8 @@ const voices = {
 // Constants
 
 const messagesMaxCharacters = 20000
+const API_KEY = 'ce69df07b50e7179cbbfc5c2bef9d752';
```

Comment:
Can we please convert these to secrets and not put them in the public IDE (or in github) 🤣